wiki/content/graphql/queries/persistent-queries.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "Persistent Queries"
+weight = 3
+[menu.main]
+    parent = "graphql-queries"
++++
+
+Dgraph supports Persistent Queries. When persistent queries are enabled, the client only sends the hash of a query to the server. The server has a list of known hashes and uses the related query accordingly. 
+
+Persistent queries also improve the performance and the security of an application.
+
+{{% notice "note" %}}
+Once stored, Queries and Hash-Keys will persist until they are cleared or overwritten.
```

Comment:
Don't have a mechanism to clear queries right now. Remove this.

Reply:
Done